packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -102,6 +102,26 @@ export interface RemoteVideoTileMenuOptions {
    */
   isHidden?: boolean;
 }
+
+/* @conditional-compile-remove(click-to-call) */
+/**
+ * Options for the local video tile in the Call composite.
+ *
+ * @beta
+ */
+export interface LocalVideoTileOptions {
+  /**
+   * Position of the local video tile. If unset will render the local tile in the floating local position.
+   *
+   * @defaultValue 'floating'
+   * @remarks 'grid' - local video tile will be rendered in the grid view of the videoGallery.
+   * 'floating' - local video tile will be rendered in the floating position and will observe overflow gallery.
```

Comment:
nit remove fullstop as sentence continues on next line?
```suggestion
   * 'floating' - local video tile will be rendered in the floating position and will observe overflow gallery
```